protos/core.proto

```diff
@@ -223,6 +223,7 @@ message ColumnDescriptor {
 
 message Table {
   Target target = 4;
+  string canonical_target = 32;
```

Comment:
what do you think about calling this `id` everywhere? that's effectively what it is

Reply:
One thing from our conversation I don't think was made clear - we do need to decode this value, it won't always be one way. It makes me a bit uneasy having code that decodes what is apparently just an ID as a structured object, at least canonical_target implies what it is, or maybe it should actually be encoded_canonical_target